src/app/modules/app/pages/users/users.component.ts

```diff
@@ -24,16 +29,19 @@ export class UsersComponent {
 
   public isSidenavOpen = true;
   public isLoading = true;
-
+  userRoles: string[] | undefined;
   columns = userTypeColumns;
   commonColumns = userCommonColumns;
+  trainerColumns = userTrainerColumns;
   users: UserFlat[] = [];
 
   constructor(
     private readonly dialogService: DialogService,
     private readonly userService: UserService,
+    private readonly keycloakService: KeycloakService,
     public readonly snackBar: MatSnackBar
   ) {
+    this.userRoles = keycloakService.getUserDetails()?.roles;
```

Comment:
If we do not have more than 1 role per user, we can have something like `keycloakService.getRole()` in the service and validate this